src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age !== undefined && <p className="Person__age">I am {age}</p>}
+      <p className="Person__partner">
+        {isMarried === true && sex === 'm' ? `${partnerName} is my wife` : null}
+        {isMarried === true && sex === 'f'
+          ? `${partnerName} is my husband`
+          : null}
+        {isMarried === false && 'I am not married'}
```

Comment:
- You don't need to compare with true or false, `isMarried` is already a boolean.
- Also as previous mentor mentioned it would be a good idea to create a variable for `wife/husband` partner status and use this variable here